tutorials/MD/04_Trajectory_Analysis.ipynb

```diff
@@ -0,0 +1,265 @@
+    "Luckily MDAnalysis has its own built-in function to calculate this and we can import it..\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import built-in MDAnalysis tools for alignment and RMSD.\n",
+    "import MDAnalysis as mda\n",
+    "from MDAnalysis.analysis import align\n",
+    "from MDAnalysis.analysis.rms import RMSD as rmsd\n",
+    "\n",
+    "# Define the simulation universe and the reference structure (protein structure at first frame)\n",
+    "protein = mda.Universe('./run/md.gro', './run/md.xtc')\n",
+    "protein_ref = mda.Universe('./run/em.gro', './run/md.xtc')\n",
+    "protein_ref.trajectory[0]\n",
+    "\n",
+    "# Call the MDAnalysis align function to align the MD simulation universe to the reference (first frame) universe\n",
+    "align_strucs = align.AlignTraj(protein, protein_ref, select=\"backbone\", weights=\"mass\", in_memory=True, verbose=True)\n",
+    "\n",
+    "R = align_strucs.run()\n",
+    "rmsd_data = R.rmsd\n",
+    "\n",
+    "# Plot the RMSD\n",
+    "pyplot.plot(rmsd_data)\n",
+    "pyplot.title(\"RMSD over time\")\n",
+    "pyplot.xlabel(\"Frame number\")\n",
+    "pyplot.ylabel(\"RMSD (Angstrom)\")\n",
+    "pyplot.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Questions**\n",
+    "* What does this tell you about the stability of the protein? Is it in a state of equilibrium and if so why and at what time?\n",
+    "* Can you think of a situation where this approach might not be a very good indication of stability?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Root mean square fluctuation (RMSF) of 1HSG**\n",
+    "\n",
+    "A similar property that is particularly useful is the root mean square fluctuation (RMSF), which shows how each residue flucuates over its average position.\n",
+    "\n",
+    "The RMSF for an atom, $\\rho_i$, is given by:\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\rho_i = \\sqrt{\\sum^N_{i=1} \\big\\langle(\\mathbf{x}_i - \\langle \\mathbf{x}_i \\rangle )^2 \\big\\rangle }\n",
+    "\\end{equation}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from MDAnalysis.analysis.rms import RMSF as rmsf\n",
+    "\n",
+    "# Define again the simulation universe, using however the renumbered .gro file that you had generated earlier\n",
+    "protein = mda.Universe('./run/em_renumbered.gro', './run/md.xtc')\n",
+    "\n",
+    "# Reset the trajectory to the first frame\n",
+    "protein.trajectory[0]\n",
+    "\n",
+    "# We will need to select the alpha Carbons only\n",
+    "calphas = protein.select_atoms(\"name CA\")\n",
+    "rmsf_calc = rmsf(calphas, verbose=True).run()\n",
+    "\n",
+    "#Plot the RMSF\n",
+    "pyplot.plot(calphas.resids, rmsf_calc.rmsf, color='darkorange' )\n",
```

Comment:
rather than residues, you should be able to use resindices+1 or maybe resindex, to get a "continuous residue number starting from 1".

Reply:
Fixed